### ISSUE TITLE
Spec: How does this spec apply to other runtimes?

### ISSUE THREAD

**Issue by jdolitsky** [solo-io/wasm#173]
I noticed that the config contains a `type` field, with `envoy_proxy` as the example used. What if I am targeting an alternative runtime such as [wasmtime](https://github.com/bytecodealliance/wasmtime)? Does the `abiVersions` apply to other runtimes, or should this be nested under `config` ?

It seems like this is targeting Envoy filters.. is it up to runtimes using this spec to determine an appropriate unique key to use for the `type` field?

**Comment by yuval-k** [comment]
yes, type may be `wasmtime`; abiVersion is meant to track the version of the runtime.
Currently envoy is our only example, but we are hoping to document more  types as we encounter more use cases.

cc @ilackarms
